app/views/user_journey/before_you_start.html.erb

```diff
@@ -27,12 +27,19 @@
 <% else %>
   <p><%= t 'user_journey.before_you_start.rotate_certificate_read_steps' %></p>
   <p><%= t "user_journey.before_you_start.#{@certificate.component.type}", type: @certificate.usage %>
-  <% if @certificate.signing? %>
+  <% if @certificate.signing? && @certificate.component.type == COMPONENT_TYPE::MSA_SHORT %>
+    <%= t 'user_journey.before_you_start.you_must' %>
+    <ul class="govuk-list govuk-list--number">
+      <li><%= t('user_journey.before_you_start.create_key_and_certificate', type: @certificate.usage) %></li>
+      <li><%= t('user_journey.before_you_start.add_signing_key_and_certificate_as_secondary', component: @certificate.component.display) %></li>
+      <li><%= t('user_journey.restart_component', component: @certificate.component.display) %></li>
+    </ul>
```

Comment:
This is all duplicated in the else just below, any way this can be restructured?